vtex/faststore
# Feat: Next 13 - Global Section - migrate `Navbar` Section

## What's the purpose of this pull request?

This PR aims to add the initial structure from the global sections and migrate the `Navbar` section from the pages router to the app router (next 13) along with its related components (Search). 

Some components were moved from the `src` folder to the `app` folder to make it Server Components, with a few updates to make them use the updated path. 

<img width="1289" alt="Screenshot 2024-06-10 at 20 28 43" src="https://github.com/vtex/faststore/assets/11325562/88bb6147-6813-4d5c-a881-10bd666469f3">


## How to test it?

- Run the local server (`yarn dev` in core package);
- Make sure that all the current pages are working as expected (`/`, `/office` and some `/<slug>/p`);
- Make sure `/fs-next-update` is rendering with thew `Navbar` section on the top of the page and without errors.

### Starters Deploy Preview

TBD
